.prettierrc.js

```diff
@@ -4,9 +4,10 @@ module.exports = {
   plugins: ['prettier-plugin-ember-template-tag'],
   overrides: [
     {
-      files: '*.{js,ts,gjs,gts}',
+      files: '*.{js,gjs,ts,gts,mjs,mts,cjs,cts}',
       options: {
         singleQuote: true,
+        templateSingleQuote: false,
```

Comment:
Would it be possible to make this `true`? I imagine it would reduce the size of the PR by a bit